test/opf/opf.jl

```diff
@@ -62,6 +68,10 @@ include("quad_obj.jl")
     @testset _test_socwr_DualSolFormat()
 
     @testset _test_socwr_DualFeasibility()
+
+    @testset _test_sdpwrm_DualSolFormat()
```

Comment:
This test looks at two entries of the dual solution, corresponding to thermal constraints.

Reply:
It's similar to the tests for SOC: https://github.com/AI4OPT/OPFGenerator/blob/2bad06c9cb4eb996e18624408ee7655f0dcbb191/test/opf/socwr.jl#L193-L211

Reply:
My point was more that, in the SDP-related unit tests, I would expect to (at least) check something that's related to SDP